.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+## TODO here we will need a cronjob.
+
+jobs:
+  prerelease:
+    name: Create prerelease
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.next-version }}
+      image-tag: ${{ steps.image-tag.outputs.tag }}
+    steps:
+      - name: Check is triggered for default branch
+        run: |
+          if [ ${GITHUB_REF_NAME} != ${{ github.event.repository.default_branch }} ] || [ ${GITHUB_REF_TYPE} != "branch" ];then
+            echo This workflow should only be triggered for the '${{ github.event.repository.default_branch }}' branch
+            exit 1
+          fi
```

Comment:
Can we have a `input` for this workflow that is called `force-out-of-branch` (that defaults to `false`) so we are able to test things out of `main`?

Reply:
Moreover, notice that if needed we will be always able to create manually tag from any branch we want the only issue is that the release notes will be not automatized

Reply:
> I would say that to test it is better to use a fork since this creates a pre-release. Unless you think it is a strong requirement I believe it is better to avoid having it

I do not see a strong requirement and I can avoid it. I am also the only one thinking about this.

> Moreover, notice that if needed we will be always able to create manually tag from any branch we want the only issue is that the release notes will be not automatized

But this is not a Pro but a Con. If we have to test something, it would be better for us than the CI is able to run it so a human does not make a human mistake.

Nevertheless, I think the first point wins.